native/cocos/bindings/auto/jsb_spine_auto.cpp

```diff
@@ -15095,7 +15098,9 @@ bool js_register_spine_SkeletonAnimation(se::Object* obj) // NOLINT(readability-
     cls->defineFunction("setTrackEventListener", _SE(js_spine_SkeletonAnimation_setTrackEventListener));
     cls->defineFunction("setTrackInterruptListener", _SE(js_spine_SkeletonAnimation_setTrackInterruptListener));
     cls->defineFunction("setTrackStartListener", _SE(js_spine_SkeletonAnimation_setTrackStartListener));
+#if SCRIPT_ENGINE_TYPE != SCRIPT_ENGINE_QUICKJS
     cls->defineFunction("ctor", _SE(js_spine_SkeletonAnimation_ctor));
+#endif
```

Comment:
Modify tools/spine.ini, 
Make `classes_need_extend` empty.
It's used in Creator 2.x.
Since Creator 3.x use typescript for writing engine code, there is no need to have `ctor` stuff now.

Reply:
I think it is better to create another issue for it.

Reply:
spine doesn't use the ctor functionality. so the toos/spine.ini doesn't need to set `SkeletonAnimation` to classes_need_extend section.
I think it's old code and not deleted in 3.x.

I create an issue here: https://github.com/cocos/cocos-engine/issues/11091